### DISCUSSION TITLE
React 18 was release today! Remix is to support React 18 soon? When?

### DISCUSSION THREAD

**Discussion by pagebanc** [remix-run/remix#2549] [Q&A] [answered]
Hello,
Long awaited React 18 is finally here (03/29/2022).
Does anyone know when Remix will support React 18 so that `npx create-remix` creates a Remix application structure using **React 18**?
Thank you!

**Comment by MichaelDeBoey** [comment, marked as answer]
Remix is already React v18 compatible (see https://twitter.com/remix_run/status/1508841826685358080).

Templates will be updated once https://github.com/remix-run/remix/pull/2540 is merged.

**Reply by pagebanc** [reply]
Thanks!
